new-dti-website/components/apply/RoleDescription.tsx

```diff
@@ -103,7 +103,15 @@ const RoleDescriptions = () => {
             Apply now
           </Link>
         ) : (
-          <></>
+          <Link
+            key="Apply Page"
+            href="#"
+            className="primary-button opacity-50 cursor-not-allowed"
+            onClick={(e) => e.preventDefault()}
+            aria-disabled="true"
+          >
+            Apply now
+          </Link>
```

Comment:
On your video, at `0.02s`, hovering on the "Apply now" button triggers a background color change. Can we remove that since the button is disabled? Same for the second button at `0.06s`

Reply:
https://github.com/user-attachments/assets/40368545-073d-497d-821f-caa69c63ae6b